addons_rksv/pos_rksv/models/res_company.py

```diff
@@ -10,6 +10,7 @@ class Company(models.Model):
     _name = 'res.company'
     _inherit = 'res.company'
 
+    rksv_at = fields.Boolean('RKSV AT', default=True)
```

Comment:
Force companies in Austria in order to not allow easily to disable the RKSV checkbox (i.e. via constraint)